pkg/features/kube_features.go

```diff
@@ -125,6 +125,12 @@ const (
 	// Add priority to pods. Priority affects scheduling and preemption of pods.
 	PodPriority utilfeature.Feature = "PodPriority"
 
+	// owner: @resouer @denkensk
```

Comment:
There used to be a comment that there should be only one owner :)

Reply:
Sure, I can own this then.